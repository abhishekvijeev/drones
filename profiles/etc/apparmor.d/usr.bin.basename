# Last Modified: Tue Aug 13 12:34:37 2019
#include <tunables/global>

/usr/bin/basename flags=(complain){
  #include <abstractions/base>

  /lib/x86_64-linux-gnu/ld-*.so mr,
  /usr/bin/basename mr,
  /lib/x86_64-linux-gnu/ld-2.29.so	r,
  /etc/ld.so.cache	r,
  /lib/x86_64-linux-gnu/libc-2.29.so	r,
  /home/abhishek/.local/share/gvfs-metadata/**	r,
  /usr/share/code/**	r,
  /dev/**	wr,
  

}
