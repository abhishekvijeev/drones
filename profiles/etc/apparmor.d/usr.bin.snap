# Last Modified: Tue Aug 13 11:55:18 2019
#include <tunables/global>

/usr/bin/snap flags=(complain) {
 	#include <abstractions/base>

 	/lib/x86_64-linux-gnu/ld-*.so mr,
 	/proc/version	r,
	/usr/bin/snap mr,
	/usr/lib/os-release	r,
	

}
